experiments/EclipseExample/src/test/java/edu/washington/cs/dt/impact/ExampleDT/CalculatorTestCase2.java

```diff
@@ -0,0 +1,71 @@
+package edu.washington.cs.dt.impact.ExampleDT;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import edu.washington.cs.dt.impact.ExampleDT.Calculator;
+
+public class CalculatorTestCase2 {
+	/* ALL BASED ON PRIORITIZATION STATEMENT ABSOLUTE
+	 * This test class represents case 2: where getSumTest is the dependent test and depends on only isEvenTest to come after it
```

Comment:
https://docs.google.com/document/d/1ddsI5ahHL-jggPKtwdRLmch8RnU9KWjH2f3afDUW1bY/edit?usp=sharing
Add this link to comments